pkg/scheduler/backend/queue/active_queue.go

```diff
@@ -204,6 +208,16 @@ func (aq *activeQueue) pop(logger klog.Logger) (*framework.QueuedPodInfo, error)
 	aq.schedCycle++
```

Comment:
Do we want to increment the scheduling cycle even if we run the `unlockedPop` second time (when the issue happens)?

Reply:
This is great catch..